pyhmy/rpc/account.py

```diff
@@ -0,0 +1,204 @@
+from .request import (
+    _default_endpoint,
```

Comment:
don't import privates, maybe use a common file, or make public

Ditto for similar imports in this PR